app/Services/CampaignService.php

```diff
@@ -62,17 +63,16 @@ public function getCampaignIdsFromSignups()
      */
     public function getPostTotals($campaign)
     {
-        return DB::table('posts')
-                ->select('campaign_id',
-                    DB::raw('SUM(status = "accepted") as accepted_count'),
-                    DB::raw('SUM(status = "pending") as pending_count'),
-                    DB::raw('SUM(status = "rejected") as rejected_count'))
-                ->whereIn('status', ['accepted', 'pending', 'rejected'])
-                // @TODO: if we ever want to expose other post types in Rogue to review/count, update this.
-                ->whereIn('type', ['photo', 'text'])
-                ->where('campaign_id', '=', $campaign['id'])
-                ->whereNull('deleted_at')
-                ->first();
+        return Post::select(
+                'campaign_id',
+                DB::raw('SUM(status = "accepted") as accepted_count'),
+                DB::raw('SUM(status = "pending") as pending_count'),
+                DB::raw('SUM(status = "rejected") as rejected_count'))
+            ->whereIn('status', ['accepted', 'pending', 'rejected'])
+            ->reviewable()
+            ->where('campaign_id', '=', $campaign['id'])
+            ->whereNull('deleted_at')
```

Comment:
Can you double-check if this is necessary now? By default, Eloquent [should exclude](https://laravel.com/docs/5.7/eloquent#querying-soft-deleted-models) soft-deleted records.

Reply:
@DFurnes ah you're right! Just got rid of it :)